code/fundamentals.rb

```diff
@@ -7,9 +7,9 @@
 puts message
 
 # 1. Create a variable `my_favorite_animal` and assign it the name of your favorite animal
-
+my_favorite_animal="horse"
 # 2. Print "My favorite animal is <your-favorite-animal>" by printing a string with `my_favorite_animal` interpolated into it.
-
+"My favorite animal is #{my_favorite_animal}"
 # 3. Write a method `make_noise` which prints the string "BANG!"
-
+make_noise="bang!" make_noise.upcase = "BANG!"
```

Comment:
This question called to make a method (function) which prints the string when invoked. Something like this:

```ruby
def make_noise
   puts "BANG!"
end
```